libioc/Network.py

```diff
@@ -397,67 +375,42 @@ def __configure_firewall(self, mac_address: str) -> typing.List[str]:
                         f"Firewall permit not possible for address '{address}'"
                     )
                     continue
-                self.queuing_firewall.add_rule(firewall_rule_number, [
+                self.firewall.add_rule(firewall_rule_number, [
                     "allow", protocol,
                     "from", _address, "to", "any",
                     "layer2",
                     "MAC", "any", mac_address,
-                    "via", f"{self._escaped_nic_name}:$IOC_JID:b",
+                    "via", f"{self._escaped_nic_name}:{self.jail.jid}:b",
                     "out"
-                ], insecure=True)
```

Comment:
what does (the removal of) this (parameter) mean?

Reply:
We no longer need a shell (that resolves env variables). We know the jails JID at the time of running this code now, so that there is simply no need for it.